mod/wikirate_source/spec/set/type_plus_right/source/file-import_spec.rb

```diff
@@ -2,191 +2,185 @@
-      expect(html).to have_tag("table", :with=>{:class=>"import_table"}) do
-        with_tag "tr" do
-          with_tag "input", :with=>{:type=>"checkbox",:value=>"43",:id=>"metric_values_Cambridge_"}
-          with_tag "td",:text=>"Cambridge"
-          with_tag "td" do
-            with_tag "input", :with=>{ :type=>"text", :name=>"corrected_company_name[Cambridge]" }
+      expect(html).to have_tag("input", with: {id: "is_metric_import_update",value: "true",type: 'hidden'})
+      expect(html).to have_tag("table", with: {class: 'import_table'}) do
+        with_tag 'tr' do
+          with_tag "input", with: {type: "checkbox",value: "43",id: 'metric_values_Cambridge_'}
+          with_tag "td",text: 'Cambridge'
+          with_tag 'td' do
+            with_tag "input", with: { type: "text", name: 'corrected_company_name[Cambridge]' }
           end
         end
-
-        with_tag "tr" do
-          with_tag "input", :with=>{:checked=>"checked",:type=>"checkbox",:value=>"9",:id=>"metric_values_Amazon.com__Inc._"}
-          with_tag "td",:text=>"Amazon.com, Inc."
-          with_tag "td",:text=>"amazon.com"
-          with_tag "td",:text=>"alias"
-          with_tag "td" do
-            with_tag "input", :with=>{ :type=>"text", :name=>"corrected_company_name[Amazon.com, Inc.]" }
+        with_tag 'tr' do
+          with_tag "input", with: {checked: "checked",type: "checkbox",value: "9",id: 'metric_values_Amazon.com__Inc._'}
+          with_tag "td",text: 'Amazon.com, Inc.'
+          with_tag "td",text: 'amazon.com'
+          with_tag "td",text: 'alias'
+          with_tag 'td' do
+            with_tag "input", with: { type: "text", name: 'corrected_company_name[Amazon.com, Inc.]' }
           end
         end
-
-        with_tag "tr" do
-          with_tag "input", :with=>{:checked=>"checked",:type=>"checkbox",:value=>"62",:id=>"metric_values_Apple_Inc._"}
-          with_tag "td",:text=>"Apple Inc."
-          with_tag "td",:text=>"exact"
+        with_tag 'tr' do
+          with_tag "input", with: {checked: "checked",type: "checkbox",value: "62",id: 'metric_values_Apple_Inc._'}
+          with_tag "td",text: 'Apple Inc.'
+          with_tag "td",text: 'exact'
         end
-        with_tag "tr" do
-          with_tag "input", :with=>{:checked=>"checked",:type=>"checkbox",:value=>"33",:id=>"metric_values_Sony_Corporation_"}
-          with_tag "td",:text=>"Sony Corporation"
-          with_tag "td",:text=>"Sony C"
-          with_tag "td",:text=>"partial"
-          with_tag "td" do
-            with_tag "input", :with=>{ :type=>"text", :name=>"corrected_company_name[Sony Corporation]" }
+        with_tag 'tr' do
+          with_tag "input", with: {checked: "checked",type: "checkbox",value: "33",id: 'metric_values_Sony_Corporation_'}
+          with_tag "td",text: 'Sony Corporation'
+          with_tag "td",text: 'Sony C'
+          with_tag "td",text: 'partial'
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.